docs/application/dotnet/guides/multimedia/media-content.md

```diff
@@ -15,52 +15,53 @@ The media files are updated by an application (by using the classes and methods
 You can only use the Tizen.Content.MediaContent classes to manage files located in specific paths.
 
 
-> **Note**   
+> [!Note]
```

Comment:
NOTE must be capitalized here for the custom style alerts to work.
Reference: [Tizen Custom Markdown Reference](https://github.com/Samsung/tizen-docs/blob/master/styleguide/custom-style.md#alerts-note-tip-important-caution-warning)
```suggestion
> [!NOTE]
```